src/bake/DevServer.zig

```diff
@@ -7413,6 +7542,26 @@ fn readString32(reader: anytype, alloc: Allocator) ![]const u8 {
     return memory;
 }
 
+const TestingBatch = struct {
+    entry_points: EntryPointList,
+    timer: std.time.Timer,
+
+    const empty: @This() = .{
+        .entry_points = .empty,
+        .timer = undefined,
+    };
```

Comment:
Why is this needed? Seems like `undefined` should not be used here. Instead a time should be provided when initialized

Reply:
when there are no entry points, there isn't a sensible time to start this at. this is why all uses assert or handle .empty. i'll add this in a comment.